front/pages/w/[wId]/subscription/index.tsx

```diff
@@ -146,6 +146,9 @@ export default function Subscription({
         headers: {
           "Content-Type": "application/json",
         },
+        body: JSON.stringify({
```

Comment:
This is the subscription from the subscription page for old users in `FREE_TEST_PLAN`.